<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VakioViiteDlg</class>
 <widget class="QDialog" name="VakioViiteDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vakioviite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Vakioviitteellä voit automatisoida saapuvan viitemaksun tiliöinnin.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="numerointiGroup">
     <property name="title">
      <string>Numerointi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="seuraavaRadio">
        <property name="text">
         <string>Seuraava viitenumero</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="valitseRadio">
        <property name="text">
         <string>Valitse numero itse</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="viiteLabel">
       <property name="text">
        <string>Viitenumero</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="viiteEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="varoKuva">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../pic/pic.qrc">:/pic/varoitus.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="varoLabel">
         <property name="text">
          <string>Viitenumero on jo käytössä</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Otsikko</string>
       </property>
       <property name="buddy">
        <cstring>otsikkoEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="otsikkoEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tili</string>
       </property>
       <property name="buddy">
        <cstring>tiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="TilinvalintaLine" name="tiliEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="kohdennusLabel">
       <property name="text">
        <string>Kohdennus</string>
       </property>
       <property name="buddy">
        <cstring>kohdennusCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KohdennusCombo" name="kohdennusCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KohdennusCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/kohdennuscombo.h</header>
  </customwidget>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VakioViiteDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VakioViiteDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
